src/Route.jsx

```diff
@@ -11,15 +11,24 @@ const ICON_MAPPING = {
   },
 };
 
-export function getStopMarkersLayer(route, getStopInfo) {
+/* changes color of selected stop along a route */
+function selectedStopColor(stop, selectedStops) {
+  const selectedStopsIds
+  = selectedStops.map(currentstop => currentstop.sid);
+  if (selectedStopsIds.includes(stop.sid)) {
+    return [0, 255, 0];
```

Comment:
@EddyIonescu is it OK to hardcode color?
Should we have an option to change icon as well instead of just color?

Reply:
This is fine for now. We can add CSS styles in the future.